src/winetricks

```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
     w_set_winver 'default'
 }
 
+w_metadata wmp11 dlls \
+    title="Windows Media Player 11" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="wmp11-windowsxp-x64-enu.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Windows Media Player/wmplayer.exe"
+
+load_wmp11()
+{
+    #w_package_unsupported_win64
+
+    # FIXME: what versions of Windows are really bundled with wmp10?
+    #w_skip_windows wmp10 && return
+
+    # See https://appdb.winehq.org/appview.php?iVersionId=3212
+    w_call wsh57
+
+    #w_call gdiplus
+
+    # http://down.cdhaha.net/?p=39
+    w_download https://web.archive.org/web/20190512112704/https://download.microsoft.com/download/3/0/8/3080C52C-2517-43DE-BDB4-B7EAFD88F084/wmp11-windowsxp-x64-enu.exe 5af407cf336849aff435044ec28f066dd523bbdc22d1ce7aaddb5263084f5526
+
+    w_set_winver winxp
+
+    # remove builtin placeholders to allow update
+    rm -f "$W_SYSTEM64_DLLS"/wmvcore.dll "$W_SYSTEM64_DLLS"/wmp.dll
+    rm -f "$W_PROGRAMS_UNIX/Windows Media Player/wmplayer.exe"
+    # need native overrides to allow update and later checks to succeed
+    w_override_dlls native l3codeca.acm wmp wmplayer.exe wmvcore wmpnssci.dll
+
+    # Crashes on exit, but otherwise ok; see https://bugs.winehq.org/show_bug.cgi?id=12633
+    w_try_cd "$W_CACHE/$W_PACKAGE"
+    # https://docs.microsoft.com/en-us/windows/win32/wmp/redistributing-windows-media-player-11
+    "$WINE" wmp11-windowsxp-x64-enu.exe /q:A /c:"setup_wm.exe /Q /R:N"
```

Comment:
Add `w_try`.

This also unconditionally installs in quiet mode. Those flags should only be passed if W_OPT_UNATTENDED is set.

Reply:
>Add w_try.

Installer tries to install some incompatible things and fails with 126 code at end.

Reply:
Ok. Please add a comment.